citrination_client/models/columns/column_factory.py

```diff
@@ -0,0 +1,37 @@
+from citrination_client.models.columns import *
+from citrination_client.base import CitrinationClientError
+
+def _flatten_column_dict(response_dict):
+    flat_dict = {}
+    for k in response_dict:
+        if k == "options" or k == "type":
+            continue
+        flat_dict[k] = response_dict[k]
+
+    if response_dict["options"]:
+        for option in response_dict["options"]:
+            flat_dict[option] = response_dict["options"][option]
+
+    return flat_dict
+
+def _get_column_class_from_type(type):
+    if type == "Real":
+        return RealColumn
+    elif type == 'Categorical':
+        return CategoricalColumn
+    elif type == 'Alloy composition':
+        return AlloyCompositionColumn
+    elif type == 'Inorganic':
+        return InorganicChemicalFormulaColumn
+    elif type == "Vector":
```

Comment:
To avoid repeating magic strings, instead check `elif type == VectorColumn.type`